server/server.py

```diff
@@ -83,16 +85,50 @@ def admin():
 def connect(sid, environ):
   print("connect ", sid)
 
-@sio.on('admin_ping')
-def adminconnect(sid, environ):
-  table_data = defaultdict(list)
-
-  for client in CLIENTS:
-    tablenumber = CLIENTS[client]
-    table_data[tablenumber].append(client)
-  data_to_send = {'tableData' : table_data,'gameState' : GAME_STATE}
-  sio.emit('admin_return', data_to_send)
+@sio.on('hint_ping')
```

Comment:
can we stop using ping everywhere? my suggestions are: 'hint_shown', 'load_admin_data', 'admin_data', 'start_game', 'game_started'

Reply:
T_T okay...